platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/sync/SyncTests.java

```diff
@@ -589,10 +591,20 @@ private void runFallenBehindTest(
         executor.setCustomPreSyncConfiguration((c, l) -> {
             when(c.getConsensus().getMinGenerationNonAncient()).thenReturn(callerMinGen);
             when(c.getConsensus().getMaxRoundGeneration()).thenReturn(callerMaxGen);
-            c.getShadowGraph().expireBelow(callerMinGen);
+            c.getShadowGraph()
+                    .updateNonExpiredEventWindow(new NonAncientEventWindow(
+                            0 /* ignored by shadowgraph */,
+                            0 /* ignored by shadowgraph */,
+                            callerMaxGen,
```

Comment:
The original test has `callerMinGen`, but you changed it to `callerMaxGen`. Was that intentional?

Reply:
Unintentional. Fixed.